README.rst

```diff
@@ -151,6 +151,43 @@ For example, if you wanted to know what code paths query the table
 The performance record herer would include a standard Python traceback attached
 to each SQL query containing "my_table".
 
+
+``capture_operation``, if not ``None``, should be a function that takes one
+argument, the given DB or cache operation, and returns a ``bool`` indiciating if
+the operation should be recorded at all (by default, all operations are
+recorded). Not capturing some operations allows for hiding some code paths to be
+ignored in your tests, such as for ignoring database queries that would be
+replaced by an external service in production.
+
+For example, if you knew that in testing all queries within some function
+would be replaced in production by something else, you could use a
+``capture_operation`` function like so:
+
+.. code-block:: python
+
+    query_context = threading.local()
```

Comment:
I'd include the imports in the example, a reader may be unfamiliar with `threading` or `contextlib`

Reply:
What do you think about providing this as a helper function (as part of the package) rather than just as an example in the README?

Reply:
I am not sure. Most applications won't need a threading local to achieve this and can use a simple module level variable, since tests don't run in threads. I think it's better we get the basic hideability in place and if several people report needing a context manager helper we could add it then.